geom/type_polygon.go

```diff
@@ -342,17 +342,27 @@ func (p Polygon) IsValid() bool {
 	return err == nil
 }
 
-// Area gives the area of the polygon.
+// Area of a Polygon is the outer ring's area minus the areas of all inner rings.
 func (p Polygon) Area() float64 {
-	area := areaOfLinearRing(p.ExteriorRing())
+	area := math.Abs(signedAreaOfLinearRing(p.ExteriorRing()))
 	n := p.NumInteriorRings()
 	for i := 0; i < n; i++ {
-		area -= areaOfLinearRing(p.InteriorRingN(i))
+		area -= math.Abs(signedAreaOfLinearRing(p.InteriorRingN(i)))
 	}
 	return area
 }
 
-func areaOfLinearRing(lr LineString) float64 {
+// SignedArea returns a positive value only for polygons with a counterclockwise winding order.
```

Comment:
The windings of the inner and outer rings could be inconsistent (e.g. both CCW or both CW). I think that's ok for this function (it would just give a weird result in that case). What are your thoughts on clarifying that behaviour in the documentation?

Maybe something like:

```
// SignedArea gives the positive area of the polygon when the outer rings are
// wound CCW and any inner rings are wound CW, and the negative area of the
// polygon when the outer rings are wound CW and any inner rings are wound CCW.
// If the windings of the inner and outer rings are the same, then the area
// will be inconsistent.
```